env.yml

```diff
@@ -46,9 +43,15 @@ dependencies:
   # chemistry
   - rdkit
   - datamol >=0.10
-  - mordred
 
-   # Dev
+  # Optional IPU
+  - torchvision
```

Comment:
I don't think that's needed anymore, see PR #386

Reply:
It's still at https://github.com/datamol-io/graphium/blob/e2394033d1a5ea14000fa6eef278b22c14714ec0/graphium/ipu/ipu_simple_lightning.py#L9

I will remove it assuming the other PR will take care of it.